apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/SpannerDbTypeTests.ValueConversions.cs

```diff
@@ -209,8 +209,8 @@ public static IEnumerable<object[]> GetValidValueConversions()
             yield return new object[] {(ushort) 11, SpannerDbType.Bool, "true", TestType.ClrToValue };
             yield return new object[] {(ushort) 0, SpannerDbType.Bool, "false"};
 
-            //Spanner type = String tests.
-            //Note the casing on bool->string follows c# bool conversion semantics (by design).
+            // Spanner type = String tests.
+            // Note the casing on bool->string follows c# bool conversion semantics (by design).
```

Comment:
nit: cas_t_ing

Reply:
Nope, it's deliberately casing here. It's because the string values are "True" and "False", not "true" and "false".

Reply:
Oh, casing as in upper case, yep.